lib/cli/commands/init.js

```diff
@@ -1,44 +1,34 @@
 // Init
-const init = {
+const initCommand = {
 	command: "init",
 	describe: "Initialize the UI5 Build and Development Tooling configuration for an application or library project.",
 	middlewares: [require("../middlewares/base.js")]
 };
 
-init.handler = async function(argv) {
-	const init = require("../../init/init");
-	const {promisify} = require("util");
-	const path = require("path");
-	const fs = require("fs");
-	const stat = promisify(fs.stat);
-	const writeFile = promisify(fs.writeFile);
-	const safeDumpYaml = require("js-yaml").safeDump;
-
-	async function exists(filePath) {
-		try {
-			await stat(filePath);
-			return true;
-		} catch (err) {
-			// "File or directory does not exist"
-			if (err.code === "ENOENT") {
-				return false;
-			} else {
-				throw err;
-			}
-		}
-	}
+initCommand.lazyRequireDependencies = async function() {
```

Comment:
What about using [mock-require](https://github.com/boblauer/mock-require) here instead of adding this function to the productive code just for resting purposes?

Reply:
As discussed, module-require is now used to mock module loading